ci.yml

```
@@ -0,0 +1,11 @@
+module: gatekeeper
+type: nodejs
+runTests: false
+testCommand: npm test
+docker: true
+securityScan: true
+publishNpm: false
```

```suggestion
publishNpm: true
```
We used it only as a lib, so publishing to nexus make sense.